drivers/src/persistent.rs

```diff
@@ -600,6 +569,25 @@ impl PersistentDataAccessor {
     }
 }
 
+#[cfg(feature = "runtime")]
+#[repr(C)]
+#[derive(IntoBytes, TryFromBytes, KnownLayout, Zeroize)]
+pub struct DpePersistentData {
```

Comment:
Do you want to make this a part of a `RuntimePersistentData`?

I'm thinking something like:

```
persistent_data
├── rom
└── runtime
    └── dpe
```

That would prevent mixing ROM and Runtime produced data

Reply:
Yeah, that is my end goal. I wanted to move this now as a cleanup before the larger separation